source/languages/en/riak/dev/using/2i.md

```diff
@@ -596,6 +722,29 @@ riak.RiakError: "Riak 2i fields must end with either '_bin' or '_int'."
 // those names
 ```
 
+```javascript
+var cmd = new Riak.Commands.KV.SecondaryIndexQuery.Builder()
+    .withBucketType('indexes')
+    .withBucket('people')
+    .withIndexName('field2_foo')
+    .withIndexKey('jsmith123')
+    .withCallback(query_cb)
+    .build();
+client.execute(cmd);
+
+// Produces the following stack trace (truncated):
+error: query_cb err: 'Error processing incoming message: error:function_clause:[{riak_api_pb
+_server,
```

Comment:
Should this be a newline, or is it supposed to be `error:function_clause:[{riak_api_pb_server,`?